Description: We install to /usr/bin, and the desktop files explicitly point to /usr/local
Author: Unit 193 <unit193@ninthfloor.org>
Origin: vendor
Last-Update: 2014-05-09

--- xfdashboard-0.1.91.orig/data/xfdashboard-autostart.desktop.in
+++ xfdashboard-0.1.91/data/xfdashboard-autostart.desktop.in
@@ -2,8 +2,8 @@
 Type=Application
 _Name=Xfdashboard
 _Comment=Start xfdashboard as daemon in background
-TryExec=/usr/local/bin/xfdashboard -d
-Exec=/usr/local/bin/xfdashboard -d
+TryExec=/usr/bin/xfdashboard -d
+Exec=/usr/bin/xfdashboard -d
 OnlyShowIn=XFCE;
 Terminal=false
 Hidden=true
--- xfdashboard-0.1.91.orig/data/xfdashboard.desktop.in
+++ xfdashboard-0.1.91/data/xfdashboard.desktop.in
@@ -2,7 +2,7 @@
 Type=Application
 _Name=Xfdashboard
 _Comment=Maybe a Gnome shell like dashboard for Xfce
-TryExec=/usr/local/bin/xfdashboard
-Exec=/usr/local/bin/xfdashboard
+TryExec=/usr/bin/xfdashboard
+Exec=/usr/bin/xfdashboard
 OnlyShowIn=XFCE;
 Terminal=false
